backend/createInterProEntries.sh.in

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Parses the interpro entries
+
+set -o pipefail -e
+exec 3>&1
+
+entries="http://ftp.ebi.ac.uk/pub/databases/interpro/entry.list"
+
+lines="$(wget -q "$entries" -O -                               \
+       | grep "^IPR"                                           \
+       | sort                                                  \
```

Comment:
Minor remark: I don't think this sort is necessary? We don't really need the interpro entries to appear in order in the database, so I think this can be removed.

Reply:
Removed the sort in #d8ecf82